### ISSUE TITLE
Script terminates if a omni-epd.ini file is used

### ISSUE THREAD

**Issue by yippieyaray** [robweber/vsmp-plus#7]
Hi Rob,

I added a 'omni-epd.ini' file to tweak some image parameter. After a restart it takes a minute until the process terminates without any information written to the 'tmp/log.log' file. To narrow down the problem I a added a try-except-block with an error output to the while (line 326) in vsmp.py. 

The error log shows:
DEBUG 02/25 12:20: Rotating image 0.0
ERROR 02/25 12:20: Unexpected err=ValueError('image has wrong mode'), type(err)=<class 'ValueError'>

The 'omni-epd.ini' has the following content:
[Display]
rotate=0
flip_horizontal=False
flip_vertical=False

[Image Enhancements]
contrast=1
brightness=2
sharpness=1

I tried different movies (e.g. test.mp4 from SlowMovie project) and got always the same error.

Ray

**Comment by yippieyaray** [comment]
Hi Rob,
A try-except-block within the loop would be very helpful, because this kind of errors stops the process without an information.

e.g.
> try
>    ...
>   except BaseException as err:
>         logging.error(f"Unexpected {err=}, {type(err)=}")
>         raise

**Comment by robweber** [comment]
Some of the image enhancement operations expect the Pillow Image to be in certain modes. I was able to recreate this and on my install I commented out [line 207](https://github.com/robweber/vsmp-plus/blob/master/vsmp.py#L207) and things started working. 

This line is unnecessary, and probably a hold over from before `omni-epd` existed. It converts the image to a BW image prior to sending it to the EPD. `omni-epd` does this for you so it's not needed. In some cases this could even hurt you if you wanted to use a color display you'd be stuck with a BW image being sent to the display regardless. Try commenting this out on your system and let me know if it fixes it. I'll remove the line and patch the source if this is the fix.
